firefox-ios/Client/Frontend/TrackingProtection/CertificatesHandler.swift

```diff
@@ -35,7 +35,7 @@ class CertificatesHandler {
             } catch {
                 DefaultLogger.shared.log("\(error)",
                                          level: .warning,
-                                         category: .homepage)
+                                         category: .certificate)
```

Comment:
these seemed more aligned with a separate category than homepage, so created a new one here. There are other areas that I feel can have its own log category, but kept it under the legacyHomepage for now.